### PR TITLE
feat(examples): add password strength checker example

### PR DESCRIPTION
**Pull request by vados-cosmonic**

### CODE REVIEW

**Comment by vados-cosmonic** [comment]
NOTE: the examples are failing because we haven't released the new `wkg` stuff to `wash`...

File: examples/typescript/components/http-hello-world/docs/slow-start.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Slow Start
+
+Want to do more of the steps manually without using `wash dev`? You've come to the rihgt place.
```
Comment (ricochet) [inline examples/typescript/components/http-hello-world/docs/slow-start.md:3]:
```suggestion
Want to do more of the steps manually without using `wash dev`? You've come to the right place.
```

File: examples/typescript/components/http-hello-world/docs/slow-start.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+This command won't return (as it's the running host process), but you can view the output of the host.
+
+## Build the component
+
+To build the [component][wasmcloud-component], we can use `wash`:
+
+```console
+wash build
```
Comment (ricochet) [inline examples/typescript/components/http-hello-world/docs/slow-start.md:28]: I half expected an "even slower" joke step with the jco commands.

**Review by ricochet — APPROVED** [review]

**Comment by ricochet** [comment]
With the aided wasi helpers the example is clear. We should consider creating a TS SDK.

**Comment by vados-cosmonic** [comment]
@ricochet yeah the helpers certainly make the code much easier to understand! We absolutely should have a TS SDK, getting something set up upstream and then building on that would be fantastic.

**Review by lachieh — APPROVED** [review]

**Pull request merged**